src/coreclr/src/tools/Common/JitInterface/ThunkGenerator/gen.bat

```diff
@@ -1,2 +1,3 @@
 cd /d %~dp0
```

Comment:
Maybe have this be `pushd` and `popd` at the end, so users can run it from anywhere without needing to change back to their directory?

Reply:
Done.